### DISCUSSION TITLE
The script of simple_stacking.py

### DISCUSSION THREAD

**Discussion by zhangsanfeng86** [NVlabs/curobo#34] [Software Q&A] [answered]
Hi ,
The script of simple_stacking.py could run.
But the color of cubes is random , is possible to set the color of cube?

**Comment by balakumar-s** [comment, marked as answer]
We use this class to generate cubes: https://docs.omniverse.nvidia.com/py/isaacsim/source/extensions/omni.isaac.core/docs/index.html?highlight=stacking#stacking

It does not provide a way to color the cubes currently.
